packages/app/documents/__tests__/document.test.ts

```diff
@@ -184,16 +188,16 @@ describe('Documents', () => {
         })
 
         test('creates a new document for create or update operations', async () => {
-            const baselineCount = await db.collection('Alerts').find().count()
-            expect(await db.collection('Alerts').find().count()).toBe(baselineCount)
+            const baselineCount = await db.collection('Alerts').countDocuments()
```

Comment:
The mongo mocking library had marked `find().count()` as deprecated, so I fixed this while working in the file.